ansible/roles_ocp_workloads/ocp4_workload_windows_node/tasks/remove_workload.yml

```diff
@@ -0,0 +1,59 @@
+---
+# Implement your Workload removal tasks here
+
+# Get the OpenShift cluster ID
+- name: Get the OpenShift cluster ID
+  shell: oc get -o jsonpath='{.status.infrastructureName}' infrastructure cluster
+  register: cluster_id
+
+# Get the region for the cluster
+- name: Get the region for the cluster
+  shell: oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.placement.region}'
+  register: cluster_region
+
+# Get the first availability zone for the cluster
+- name: Get the first availability zone for the cluster
+  shell: oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.placement.availabilityZone}'
+  register: cluster_az
+
+# Get the latest Windows Server 2019 with Containers AMI
+- name: Get the latest Windows Server 2019 with Containers AMI
+  shell: >
```

Comment:
`ec2_instance` module?

Reply:
@wkulhanek 

Looks like it's missing a module

```shell
$ ansible-playbook  -i inventory  -e "ansible_user=ec2-user" -e "ansible_python_interpreter=/opt/virtualenvs/k8s/bin/python" test.yaml 

PLAY [all] ***************************************************************************************************************************************

TASK [Gathering Facts] ***************************************************************************************************************************
ok: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [k8s info extract aws access key] ***********************************************************************************************************
ok: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [convert access key into a fact for later use] **********************************************************************************************
ok: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [Extract the AWS secret key from OpenShift] *************************************************************************************************
ok: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [Convert extraced AWS secret key into a decoded fact for later use] *************************************************************************
ok: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [Remove existing credentials file] **********************************************************************************************************
changed: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [Add AWS credentials for shared account] ****************************************************************************************************
changed: [bastion.lax-9248.sandbox1854.opentlc.com]

TASK [Extract OpenShift machineSet region] *******************************************************************************************************
fatal: [bastion.lax-9248.sandbox1854.opentlc.com]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (botocore or boto3) on bastion.lax-9248.internal's Python /opt/virtualenvs/k8s/bin/python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}

PLAY RECAP ***************************************************************************************************************************************
bastion.lax-9248.sandbox1854.opentlc.com : ok=7    changed=2    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0 
```

Reply:
Would be nice to use `ec2_ami_info` ...that way I don't have to overwrite the `~/.aws/credentials` file. I can just call the extracted keys directly.

Reply:
I'll add boto3 to the virtualenv